examples/CMakeLists.txt

```diff
@@ -243,6 +243,14 @@ if(WANT_CURL_EXAMPLE)
     endif(CURL_FOUND)
 endif(WANT_CURL_EXAMPLE)
 
+# The enet example won't compile if the enet library isn't available.
+# This example isn't important so it's disabled by default to prevent problems.
+option(WANT_ENET_EXAMPLE "Build ex_enet example" off)
```

Comment:
Ideally I'd use `find_package`, but support for Enet isn't built into Cmake. I did find [this](https://github.com/stuntrally/stuntrally/blob/master/cmake/FindENet.cmake) which might work.